lib/util/merge.js

```diff
@@ -10,6 +10,14 @@ var debug = require('debug')('monocle-api:util:merge');
  * @return merged object/array
  */
 var merge = module.exports = function(/* obj1, obj2, obj3 */) {
+    function isObject(element) {
```

Comment:
This should be moved outside of the `merge` function, because it's re-declared for every merge.